packages/common/src/utils/timeFrames.ts

```diff
@@ -191,12 +191,46 @@ const databricksConfig: WarehouseConfig = {
     },
 };
 
+const trinoConfig: WarehouseConfig = {
+    getSqlForTruncatedDate: (timeFrame: TimeFrames, originalSql: string) =>
```

Comment:
Once you rebase, you will notice that now we allow the user to configure the start of the week. 
Some warehouses allow to change it via session params but for the rest, we need to handle it in the SQL. 

I haven't found any documentation about this for Trino. Only found an [open ticket](https://github.com/trinodb/trino/issues/2379) so I suspect we need to apply the SQL logic.

Reply:
Hmmm thanks I'll look how we can solve this on Trino

Reply:
@vapolonio Most warehouses don't have that feature either.
We can use the same SQL logic, inside the `getSqlForTruncatedDate`, like we do for most warehouses. ( you can see it in the main branch or when you rebase your branch )